data/core/macros/abilities.cfg

```diff
@@ -320,15 +320,17 @@ Enemy units cannot see this unit while it is in deep water, except if they have
 #enddef
 
 #define ABILITY_FEEDING
+    # technicialyl this is a weapon special, but like all weaponspecials it can nowdays be used in [abilities] aswell.
```

Comment:
Missing a space in "as well", and there are swapped letters in "technically".

Reply:
will fix, thx